apps/meteor/tests/e2e/message-actions.spec.ts

```diff
@@ -38,6 +38,34 @@ test.describe.serial('message-actions', () => {
 
 		await expect(poHomeChannel.tabs.flexTabViewThreadMessage).toHaveText('this is a reply message');
 	});
+
+	test.only('expect follow/unfollow message with thread closed', async ({ page }) => {
+		await test.step('start thread', async () => {
+			await poHomeChannel.content.sendMessage('this is a message for reply');
+			await page.locator('[data-qa-type="message"]').last().hover();
+			await page.locator('role=button[name="Reply in thread"]').click();
+			await page.locator('.rcx-vertical-bar').locator(`role=textbox[name="Message #${targetChannel}"]`).type('this is a reply message');
+			await page.keyboard.press('Enter');
+			await expect(poHomeChannel.tabs.flexTabViewThreadMessage).toHaveText('this is a reply message');
+		});
+
+		// close thread before testing because behaviour changes
```

Comment:
maybe add a test with the thread open too, since the behavior can be different?

Reply:
The difference was that with the thread open it would always work since we listen to the subscription stream in that case. In a more recent PR I have unit tests for the Thread metrics components aswell. Do you think another test is still needed for this particular PR?

Reply:
I'll do it just for the sake of it. I also forgot to remove the .only from the test so all other message-actions tests are being ignored